include/lapack/lahqr_eig22.hpp

```diff
@@ -0,0 +1,78 @@
+/// @file lahqr_eig22.hpp
+/// @author Thijs Steel, KU Leuven, Belgium
+//
+// Copyright (c) 2013-2022, University of Colorado Denver. All rights reserved.
+//
+// This file is part of <T>LAPACK.
+// <T>LAPACK is free software: you can redistribute it and/or modify it under
+// the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
+
+#ifndef __LAHQR_EIG22_HH__
+#define __LAHQR_EIG22_HH__
+
+#include <complex>
+
+#include "lapack/utils.hpp"
+#include "lapack/types.hpp"
+
+namespace lapack
+{
+
+    /** Computes the eigenvalues of a 2x2 matrix A
+     *
+     * @param[in] a00
+     *      Element (0,0) of A.
+     * @param[in] a01
+     *      Element (0,1) of A.
+     * @param[in] a10
+     *      Element (1,0) of A.
+     * @param[in] a11
+     *      Element (1,1) of A.
+     * @param[out] s1
+     * @param[out] s2
+     *      s1 and s2 are the eigenvalues of A
+     *
+     * @ingroup geev
+     */
+    template <
+        typename T,
+        typename real_t = real_type<T>>
```

Comment:
Interesting way of defining `real_t`! I will try to adopt it when dealing with `T` and `real_type<T>` on the function signature.